### PR TITLE
Because of the constant EC consumption of ECLSS

### PR DESCRIPTION
**Pull request by Gordon-Dry**
@PiezPiedPy and me fiddled this out in Discord for a few hours.

We took the Apollo 11 LM and the ISS into account to fiddle this number.

### CODE REVIEW

**Comment by PiezPiedPy** [comment]
ty 😎

**Comment by Gordon-Dry** [comment]
Not ready yet, needs some numbers fixing provided by PiezPiedPy.
And also some code changes PR from PiezPiedPy into my branch.

File: GameData/Kerbalism/Support/CryoTanks.cfg [hunk 1]
```diff
@@ -273,3 +273,14 @@ RESOURCE_DEFINITION:NEEDS[CryoTanks,Kerbalism,ProfileDefault]
 	density = 0.0
 	isVisible = false
 }
+
+// ============================================================================
+// Add the planner to all CryoTanks with cooling
+// ============================================================================
+@PART[*]:HAS[@MODULE[ModuleCryoTank]:HAS[#CoolingCost[>0]]]:HAS[!MODULE[PlannerController]]:NEEDS[CryoTanks,Kerbalism,ProfileDefault]:AFTER[zzzKerbalism]
```
Comment (PiezPiedPy) [inline GameData/Kerbalism/Support/CryoTanks.cfg]: change zzzKerbalism to zzzCryoTanks

Comment (Gordon-Dry) [reply GameData/Kerbalism/Support/CryoTanks.cfg]: ok

File: GameData/Kerbalism/System/Misc.cfg [hunk 2]
```diff
@@ -96,3 +96,18 @@
...
 
 @PART[kerbalism-container-*]:FINAL { !ContainerVolume = _ }
+
+
+// ============================================================================
+// Increase all crewable parts' EC because ECLSS uses EC constantly,
+// you wanna survive in the Apollo LM, right?
+// ============================================================================
+
+@PART[*]:HAS[#CrewCapacity[>0],@RESOURCE[ElectricCharge]]:NEEDS[FeatureHabitat]:AFTER[zzzKerbalism]
```
Comment (PiezPiedPy) [inline GameData/Kerbalism/System/Misc.cfg]: change NEEDS[FeatureHabitat] to NEEDS[ProfileDefault]

Also in the Tags section change FOR[Kerbalism] to FOR[zzzKerbalism] as in the post on Discord

**Comment by PiezPiedPy** [comment]
Re: Planner Button for Ion Engines, I will add the EngineModule into the code so it is detected by Kerbalism

File: GameData/Kerbalism/System/Misc.cfg [hunk 3]
```diff
@@ -83,16 +83,51 @@
...
+@PART[kerbalism-greenhouse]:AFTER[zzzKerbalism] { %tags = #$tags$ habitat crew greenhouse food crop hydroponic co2 carbondioxide waste atmosphere wasteatmosphere h2o water o2 oxygen }
+@PART[kerbalism-container-inline-*]:AFTER[zzzKerbalism] { %tags = #$tags$ supply supplies food water waste wastewater container }
+@PART[kerbalism-container-inline-0625]:AFTER[zzzKerbalism] { %tags = #$tags$ 625 0625 0.625 .625 }
+@PART[kerbalism-container-inline-125]:AFTER[zzzKerbalism] { %tags = #$tags$ 125 1.25 }
+@PART[kerbalism-container-inline-250]:AFTER[zzzKerbalism] { %tags = #$tags$ 250 2.5 2.50 }
+@PART[kerbalism-container-inline-375]:AFTER[zzzKerbalism] { %tags = #$tags$ 375 3.75 }
+@PART[kerbalism-*pump]:AFTER[zzzKerbalism] { %tags = #$tags$ harvest extract pump h2o water nitrogen nh3 ammonia }
+@PART[kerbalism-container-radial-*]:AFTER[zzzKerbalism] { %tags = #$tags$ external radial tank pressurized o2 oxygen nitrogen h2 hydrogen nh3 ammonia co2 carbondioxide }
+@PART[kerbalism-antenna]:AFTER[zzzKerbalism] { %tags = #$tags$ antenna }
+
```
Comment (PiezPiedPy) [inline GameData/Kerbalism/System/Misc.cfg]: use FOR[Kerbalism] on these ;)

Comment (Gordon-Dry) [reply GameData/Kerbalism/System/Misc.cfg]: Hmm. Line 89? That would overwrite all of them.

Comment (lordcirth) [reply GameData/Kerbalism/System/Misc.cfg]: Good change, but not really under "Increase-all-crewable-parts'-EC-because-ECLSS-uses-EC-constantly", is it?

Comment (Gordon-Dry) [reply GameData/Kerbalism/System/Misc.cfg]: I could open a dozen branches.
Should I?
Confuse me and everybody else as well?

However, I'm new to this.

**Comment by steamp0rt** [comment]
Ping me when this is ready

**Pull request merged**